articles/policies/legacy-rate-limits.md

```diff
@@ -0,0 +1,173 @@
+---
+description: Describes Auth0's rate limit policy for subscriptions created before 05-21-2020 when working with Auth0 API endpoints.
+topics:
+    - auth0-policies
+    - rate-limits
+contentType:
+  - reference
+useCase:
+  - support
+---
+# Managment API Endpoint Rate Limits before May 19, 2020
+
+**This policy is effective for all paid and free subscriptions made before May 19, 2020.** 
+
+::: warning
```

Comment:
Can we add the same sort of warning for Entity Limits policy? 

The policy will be enabled for self service and free subscriptions created on or after May 19, and for all self service and free subscriptions starting on June 18.

Reply:
I added the note